tests/helpers/tags.ts

```diff
@@ -0,0 +1,117 @@
+import {
```

Comment:
For all the objects you're creating here, you can return their `POJO` versions using `lean()` or `toObject()` methods. It'll also help you provide easier typescript types.